src/App.tsx

```diff
@@ -2,100 +2,14 @@
 import React from 'react';
 
 import './App.scss';
-// import moviesFromServer from './api/movies.json';
+import moviesFromServer from './api/movies.json';
+import { MoviesList } from './components/MoviesList';
```

Comment:
You should create a separate file for 'MoviesList' component, and import it into your 'App.tsx'.